docs/about.rst

```diff
@@ -45,28 +45,36 @@ From Wikipedia_:
+
+Since then, Yellowbrick has enjoyed the participation of a large number of contributors from around the world and growing support in the PyData community. Yellowbrick has been featured in talks at PyData, Scipy, NumFOCUS, and PSF organized events as well as blog posts and Kaggle competitions. We are so thrilled to have such a dedicated community involved in active contributions both large and small.
 
 For a full list of current maintainers and core contributors, please see `MAINTAINERS.md <https://github.com/DistrictDataLabs/yellowbrick/blob/develop/MAINTAINERS.md>`_ in the root of our GitHub repository. Thank you so much to everyone who has `contributed to Yellowbrick <https://github.com/DistrictDataLabs/yellowbrick/graphs/contributors>`_!
 
+Affiliations
+------------
+
+Yellowbrick is proud to be affiliated with several organizations that provide institutional support to the project. Such support is sometimes financial, often material, and always in the spirit of free and open source software. We can't thank them enough for their role in making Yellowbrick what it is today.
+
+`District Data Labs`_: District Data Labs incubated Yellowbrick and sponsors research labs by purchasing food and organizing events. Research labs are semester long sprints that allow Yellowbrick contributors to meet in person, share a meal, and hack on the project. DDL also sponsors travel to PyCon and PyData conferences for Yellowbrick maintainers and helps us buy promotional material such as stickers and t-shirts.
+
+`NumFOCUS`_: Yellowbrick is a NumFOCUS affiliated project (not a fiscally sponsored project). Our relationship with NumFOCUS has given us a lot of data science cred in the community by being listed on their website. We are also able to apply for small development grants and infrastructure support. We often participate in the project developers mailing list and other activities such as Google Summer of Code.
+
+`Georgetown University`_: Georgetown primarily provides space for Yellowbrick events including the research labs. Additionally, Georgetown Data Science Certificate students are introduced to Yellowbrick at the beginning of their machine learning education and we often perform user testing of new features on them!
+
+Yellowbrick is developed by volunteers who work on the project in their spare time and not as part of their regular work. If Yellowbrick has become critical to the success of your organization, please consider giving back to Yellowbrick. One way would be to make a financial donation to NumFOCUS on behalf of Yellowbrick. Another would be to contract DDL to give a corporate training on visual machine learning with Yellowbrick. If interested in a more formal affiliate relationship, please get in touch with us directly.
```

Comment:
Maybe worthy of an additional subhead? I also like the idea of bulletpointing the suggestions, e.g. 

How to Support Yellowbrick
----------------------------
Yellowbrick is developed by volunteers who work on the project in their spare time and not as part of their regular work. If Yellowbrick has become critical to the success of your organization, please consider giving back to Yellowbrick in one of the following ways:
 - By making a financial donation to NumFOCUS on behalf of Yellowbrick. 
 - By contracting with DDL to give a corporate training on visual machine learning with Yellowbrick. 
 - By getting in touch with us directly if interested in a more formal affiliate relationship!